akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -287,8 +286,8 @@ object HttpEntity {
     def dataBytes: Source[ByteString, Any] = data
 
     /**
-      * See [[HttpEntity#withSizeLimit]].
```

Comment:
Yes, scalariform can fix those: https://github.com/daniel-trinh/scalariform#placescaladocasterisksbeneathsecondasterisk

Perhaps we should upgrade to the latest one and enable this setting btw (not really sure what's supported in the version we're on)